_includes/partner-logo.html

```diff
@@ -1,8 +1,7 @@
 <div class="level">
-{% assign partners = site.data.partners | sample: site.data.team.size %}
-{% for partner in partners %} 
-    <a class="level-item has-text-centered" href="{{ partner.link }}">
-      <img src="assets/img/partners/{{ partner.photo }}"  alt="" style="height: 128px;">
-    </a>
-  {% endfor %}
-</div>
\ No newline at end of file
+	{% assign partners = site.data.partners | sample: site.data.team.size %} {% for partner in partners %}
```

Comment:
The `sample` part is to randomize the order right? I think for the partners it is better to hard code our preferred order. I will change this.

Reply:
Good point, thank you!